packages/apps/frontera/public/tracker-v0.3.js

```diff
@@ -27,7 +27,7 @@
 
   function track(event) {
     trackerData.activity = trackerData.activity.concat(
-      `${new Date().valueOf()},${event},`,
+      `${new Date().valueOf()},${event}`,
```

Comment:
i think this is going to produce wrong values. the solution would be a substring at the end ( substring ( 0 , len - 2 ))